### ISSUE TITLE
Lost episodes

### ISSUE THREAD

**Issue by lamaresh** [SchizoDuckie/DuckieTV#969]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Browser Action

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Stable

**What is your Operating System (Windows, Mac, Linux, Android)**
Win

**Describe the problem you are having and steps to reproduce if available**
After updating a lost all my "watched" flag on episodes and if i try to clean DuckieTV and import a backup I got all series but no episode is set as watched. I have 2 pc with Win10 and I have this problem only with one of them :\

**Comment by garfield69** [comment]
Sorry about that, it appears that for some machines the upgrade migration step has not worked as intended, so far about 11 in 3955 have reported issues :-(

Assuming that both machines have the same series and settings, then to recover I suggest the following:

- On the machine with the good DuckieTV make a backup from the *settings,backup* panel
- Then on the machine with the broken DuckieTV:
- Visit the `chrome://extensions/` on your browser,
- and use the **trash-bin** icon to remove your DuckieTV extension.
- Then shutdown your Browser completely and start it up again (this gives Chrome the chance to clean up the dataspace)
- Now install the DuckieTV extension from https://chrome.google.com/webstore/detail/cdfkaloficjmdjbgmckaddgfcghgidei
- And Start the extension
- Wait a minute or two for everything to settle down
- And at the *You have no series yet!* page, import your backup to restore your DuckieTV series and Settings.

Let me know how you go.

**Comment by garfield69** [comment]
An alternative option is that if you archive the backup you've been using on the machine with the bad DuckietV, and drag and drop the zip here, I can check that backup file to see if it contains the episode flags that you are reporting go missing during your restore.     
It may be that the backup was taken on a database that was already affected by the broken upgrade.

**Comment by lamaresh** [comment]
This backup works on previous DuckieTV version, but not on the latest.

[DuckieTV Sep 20, 2015.zip](https://github.com/SchizoDuckie/DuckieTV/files/1369457/DuckieTV.Sep.20.2015.zip)

**Comment by garfield69** [comment]
>This backup works on previous DuckieTV version, but not on the latest.      

I've tested your backup on my sandbox and I can restore all the shows and settings within the file.     
Randomly sampling the shows, I can see shows that have been completed: (the killing, the whispers), and shows partly watched: (orange s1-3 watched, s4-5 not downloaded), (sleepy hollow s1-2 watched, s3-4 not downloaded), and shows in progress (graceland, s1-2 watched, s3 watched up to 3.9, downloaded 3.10).

So the backup is good.
So I would go ahead and use this backup in the uninstall/reinstall/restore steps I described earlier.
